backend/test/integrated/group.js

```diff
@@ -163,19 +166,21 @@ describe("Groups", function () {
 				}]
 		};
 		it("with valid parameters should succeed", function(done) {
+			let groupId;
 
 			async.series([
 				function(done) {
 					agent.post(`/${username}/${model}/groups/`)
 						.send(data)
 						.expect(200 , function(err, res) {
+							groupId = res.body._id;
 							done(err);
 					});
 				},
 				function(done) {
-					agent.get(`/${username}/${model}/revision/master/head/groups/?noIssues=true`)
+					agent.get(`/${username}/${model}/groups/revision/master/head/${groupId}`)
```

Comment:
this is the deprecated api - need to swap group with revision

Also this is calling the specific group's API insetad of groupList. is this intended?

Reply:
Yes, it's intended because I think it's more useful to check the contents of the group rather than the the length of the group list array.

Reply:
Actually do you need this 2nd API request? I thought HTTP POST returns you the whole group